force-app/main/default/lwc/hot_matchingServiceAppointmentsFromSA/hot_matchingServiceAppointmentsFromSA.js

```diff
@@ -0,0 +1,87 @@
+import { LightningElement, wire, track, api } from 'lwc';
+import getServiceAppointments from '@salesforce/apex/HOT_ServiceAppointmentController.getServiceAppointments';
+
+export default class Hot_matchingServiceAppointmentsFromSA extends LightningElement {
+    @api recordId;
+    @track showList = true;
+    @track empty = false;
+    @track columns = [
+        {
+            label: 'Appointment Number',
+            fieldName: 'SAppointmentNumber',
+            type: 'url',
+            typeAttributes: { label: { fieldName: 'AppointmentNumber' }, target: '_blank' },
+            sortable: true
+        },
+        {
+            label: 'Start tid',
+            fieldName: 'SchedStartTime',
+            type: 'date',
+            sortable: true,
+            typeAttributes: {
+                day: 'numeric',
+                month: 'numeric',
+                year: 'numeric',
+                hour: '2-digit',
+                minute: '2-digit',
+                hour12: false
+            }
+        },
+        {
+            label: 'Slutt tid',
+            fieldName: 'SchedEndTime',
+            type: 'date',
+            sortable: true,
+            typeAttributes: {
+                day: 'numeric',
+                month: 'numeric',
+                year: 'numeric',
+                hour: '2-digit',
+                minute: '2-digit',
+                hour12: false
+            }
+        },
```

Comment:
Disse datetime feltene er blanke for åpne og released oppdrag. Bruk EarliestPermittedStart og DueDate istedenfor